app/src/main/java/com/simply/birthdayapp/main/MainScreen.kt

```diff
@@ -41,9 +50,19 @@ fun MainScreen(
                 AddEventScreen()
             }
             composable<BottomNavBarDestination.ProfileDestination> {
-                ProfileScreen()
+                ProfileScreen(navigateToEditAccount = {
+                    navController.navigate(ProfileDestination.EditMyProfileDestination)
```

Comment:
if you have separate ProfileDestination type why you keep it with  BottomNavBarDestination in same host, it will be more clear way to have separate NavHost for ProfileDestination